src/_test/App.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import Category from '../Pages/Category';
+import store from '../Redux/store';
+
+it('Categories renders correctly', () => {
+  const categories = renderer
+    .create(<Provider store={store}><Category /></Provider>)
+    .toJSON();
+  expect(categories).toMatchSnapshot();
+});
```

Comment:
- [ ] Excellent work with the capstone project, you have done really well :clap: , I just have one small suggestion, as mentioned in the project requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219340337-1fba08b9-9b97-4fab-9c33-90da16e683d9.png)

Please try to add more test cases to cover as much as possible for your application, at least test your pure functions in redux i.e. your reducers, rendering of each page the home page the categories page and the details page :+1: